<?xml version="1.0"?>
<!--
    This file is part of ALPHA AUV project.

    Author: Jason Miller
    Email: jason_miller@uri.edu
    Year: 2023

    Copyright (C) 2022 Smart Ocean Systems Laboratory
-->
<launch>

    <node ns="topside" name="seatrac_modem" pkg="seatrac" type="seatrac_modem"  output="screen">
        <rosparam command="load" file="$(find seatrac)/config/usbl_config.yaml"/>
        <param name="port" value="/dev/ttyUSB0" type="str" />
    </node>

</launch>